e2e/app/component/help-sidebar.ts

```diff
@@ -16,14 +16,23 @@ enum SectionSelectors {
   ModifiersForm = '//*[@id="modifiers-form"]',
   SelectionList = '//*[@id="selection-list"]',
 }
+enum TabSelectors {
+  Criteria = '//*[@data-test-id="help-sidebar-icon-criteria"]'
+}
 
 export default class HelpSidebar extends Container {
 
   constructor(page: Page, xpath: string = defaultXpath) {
     super(page, xpath);
   }
 
+  async clickTabIcon(xpath: string): Promise<void> {
+    const icon = await this.page.waitForXPath(xpath, {visible: true});
+    await icon.click();
+  }
+
   async getPhysicalMeasurementParticipantResult(filterSign: FilterSign, filterValue: number): Promise<string> {
+    await this.clickTabIcon(TabSelectors.Criteria);
```

Comment:
@dolbeew please take another look tomorrow. I'm not sure whether I'm fixing a testing idiosyncrasy or if this was a real issue

Reply:
The sidebar is _supposed_ to be open already here after [clicking the link in the tree](https://github.com/all-of-us/workbench/blob/ch/sidebar-tab/e2e/app/page/cohort-search-page.ts#L111), so clicking the criteria icon shouldn't be necessary. Not sure why it's not working, taking a look locally now.

Reply:
Ok, I see the issue now. If the sidebar is closed, it won't open when the 
![Screen Shot 2020-09-30 at 11 13 58 AM](https://user-images.githubusercontent.com/40036095/94711720-1adc4200-030e-11eb-9915-ea396104d466.png) icon is clicked now that the inactive sidebar content isn't rendered. 

It's an easy fix, but I can't suggest changes to unchanged files. Okay if I push a commit?

Reply:
Yep - go for it

Reply:
Done. Also reverted the previous commit since the test _should_ pass now.

Reply:
Passing - thanks!